package-lock.json

```diff
@@ -0,0 +1,3216 @@
+{
+  "name": "gx-it-proxy",
+  "version": "0.0.6",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {
+    "": {
+      "name": "gx-it-proxy",
+      "version": "0.0.6",
+      "license": "MIT",
+      "dependencies": {
+        "commander": "~2.19.0",
+        "eventemitter3": "3.1.0",
+        "http-proxy": "1.18.1",
+        "node-watch": "0.6.2",
+        "sqlite3": "5.1.2"
+      },
+      "bin": {
+        "gx-it-proxy": "lib/main.js",
+        "gx-it-proxy-create-db": "lib/createdb.js"
+      },
+      "devDependencies": {
+        "axios": "*",
+        "axios-retry": "*",
+        "chai": "*",
+        "eslint": "^8.3.0",
+        "eslint-config-strongloop": "^2.1.0",
+        "mocha": "*",
+        "prettier": "^3.0.3"
+      }
```

Comment:
Here, I have reverted `axios-retry` and `chai` to `"*"`. This does not break the file, it will still work. You may consider adding the version specifiers I mention in the PR's description to package.json, and it may help even further to work out what version specifiers are adequate for `axios` and `mocha`.